### PR TITLE
Adds (post): gentle introduction to distributed authorization in CanDIG

### PR DESCRIPTION
**Pull request by amanjeev**
@ljdursi This is a very simple introduction. Please review.

### CODE REVIEW

File: _posts/2020-03-04-gentle-intro-to-federated-distributed-authz.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+institutes want to control the sharing of their own resources and do not trust
+a central authority for authentication. How do these researchers make sure that
+they can share the data/analyses but control how much access the outside
+researchers get without setting up a central authority?
+
+### CanDIG as a solution
+
+This model is an extremely simplified version of the Canadian healthcare system,
+which is governed exclusively by provincial jurisdiction, and each province has
+its privacy regulations. This is where CanDIG, as a platform, helps. It is a
```
Comment (ljdursi) [inline _posts/2020-03-04-gentle-intro-to-federated-distributed-authz.md]: I'd say more of the health research environment than the healthcare system in this case; maybe something like: "This is a simplified version of the Canadian health research environment: health is under provincial jurisdiction and each province has health privacy regulations."

File: _posts/2020-03-04-gentle-intro-to-federated-distributed-authz.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+to support local control, audit access, logs while allowing API access only to
+the participating institutes for a given project. In other words, a hard
+requirement is to be fully distributed. This means that the following have to
+hold - 
+
+- No central identity
+- No central authorization authority
+- Authorization made locally based on local policies, informed by platform-wide CanDIG services
+- Site asked to provide data must be able to verify any such remotely provided information
+- Sites are completely symmetric from a Data Use & Access point of view
```
Comment (ljdursi) [inline _posts/2020-03-04-gentle-intro-to-federated-distributed-authz.md]: Can probably lose that last bullet point; Data Use & Access had a very specific meaning in the GA4GH plenary talk but might not mean anything here.

**Review by ljdursi — CHANGES_REQUESTED** [review]
I think this is a very nice introduction.  Please take a look at the couple small tweaks I've proposed at the beginning, but otherwise this is great.

**Comment by amanjeev** [comment]
@ljdursi thank you! I have made the changes.

**Review by ljdursi — APPROVED** [review]

**Pull request merged**